part1/courseinfo/src/App.jsx

```diff
@@ -0,0 +1,59 @@
+const Header = (props) => {
+  return (
+    <div>
+      <h1>Course: {props.course}</h1>
+    </div>
+  )
+}
+
+const Content = (props) => {
+  return (
+    <div>
+      <Part part={props.part1} exercises={exercises1}/>
+      <Part part={props.part2} exercises={exercises2}/>
+      <Part part={props.part3} exercises={exercises3}/>
+    </div>
+  )
+}
+
+
+const Part = (props) => {
+  return (
+    <div>
+      <p>Part: {props.part} Exercises: {props.exercises}</p>
+    </div>
+  )
+}
+
+
+const Total = (props) => {
+  return (
+    <div>
+      <p>Total Exericses: {props.total} </p>
+    </div>
+  )
+}
+
+
+const App = () => {
+  const course = 'Half Stack application development'
+  const part1 = 'Fundamentals of React'
+  const exercises1 = 10
+  const part2 = 'Using props to pass data'
+  const exercises2 = 7
+  const part3 = 'State of a component'
+  const exercises3 = 14
+
+  return (
+    <div>
+      <Header course={course} />
+      <Content 
+              part1={part1} exercises1={exercises1} 
+              part2={part2} exercises2={exercises2} 
+              part3={part3} exercises3={exercises3} />
```

Comment:
```suggestion
      <Content 
          part1={part1} exercises1={exercises1} 
          part2={part2} exercises2={exercises2} 
          part3={part3} exercises3={exercises3}
      />
```

More a style thing but good for readability and habit building